### ISSUE TITLE
Final Project Feedback

### ISSUE THREAD

**Issue by JasonC1217** [COGS118A/Group008-Wi23#3]
Total (23.5/24)

Title & Abstract (2/2)

Background (3/3)

Problem Statement (3/3)

Data (3/3)

Proposed Solution (3/3)

Metrics (3/3)

Results (3/4)
•	Comparison between different hyperparameters is needed (one easy way to do that is to include a graph) (-0.5) -Yicheng

Discussion (3/3)
•	Interpreting the result (1/1)

•	Limitations (0.5/0.5)

•	Ethics & Privacy (0.5/0.5)

•	Conclusion (1/1)

Other comments:
	Actually, you should remove beer_id from your dataset… They do not carry any information. Even if you choose to include this, it should be treated as categorical data instead of numerical data. 
	Also, for your neural network, given that you are trying to approximate a possible function, Tanh or sigmoid might be a better choice. ReLU cannot be treated as a step-function-like activation function. You might also want to consider PCA to decrease the amount of feature input into your neural network or at least you should consider normalizing your data before sending them into your neural network. I’m not going to take points off because they are beyond the scope of this course. -Yicheng
        Great job! Congrats on completing COGS 118A! -Jason
